sdk/SmartUI_SDK_local.js

```diff
@@ -0,0 +1,18 @@
+import  { Builder, By, Key, until } from 'selenium-webdriver';
+import { smartuiSnapshot } from '@lambdatest/selenium-driver';
+
+(async function example() {
+    let driver = await new Builder()
+        .forBrowser('edge')
```

Comment:
Can we use same browser for local and cloud ?

Reply:
done